azure-quantum/azure/quantum/qiskit/backends/quantinuum.py

```diff
@@ -94,9 +94,20 @@ def _azure_config(self) -> Dict[str, str]:
             }
         )
         return config
+    
+    def _basis_gates(self) -> List[str]:
+        # We use all normal QIR basis gates, with the exception of `swap`
```

Comment:
Even though the difference between the Quantinuum basis gates and the rest of the providers is just the swap gate, it would be easier to see the basis gates if it was an explicit list.